addons/web_tour/views/tour_views.xml

```diff
@@ -1,9 +1,27 @@
 <?xml version="1.0" encoding="utf-8"?>
 <odoo>
+        <record id="edit_tour_form" model="ir.ui.view">
+            <field name="model">web_tour.tour</field>
+            <field name="arch" type="xml">
+                <form string="Menu" create="0" >
```

Comment:
This one does not have an `edit="0"` for consistency with the others?
And currently it has an extra space before `>`

Reply:
thanks for the guidance, updated the pr with edit="0" and removed the extra space